src/bridge/evm/index.ts

```diff
@@ -105,7 +165,7 @@ export class EvmBridge extends ApprovalBridge {
     return new this.web3.eth.Contract(abiItem, contractAddress) as any;
```

Comment:
This is the only place where you use web3. For this case you can create empty web3 instance by yourself, so user doesn't need to pass it in this class

Reply:
I wanted to say, that for building raw transaction you do not need a connection

Reply:
It is the case for web3. But I couldn't manage to make it work for TronWeb. TronWeb still makes some requests to a node when creating a transaction.
Are you suggesting to make a parameter `provider` optional only when Web3 is required and mandatory when it has to be TronWeb?

Reply:
Maybe, need to think